core/lib/src/logger.rs

```diff
@@ -17,15 +17,27 @@ pub enum LoggingLevel {
     Normal,
     /// Shows everything.
     Debug,
+    /// Shows nothing.
+    Off,
 }
 
 impl LoggingLevel {
     #[inline(always)]
-    fn max_log_level(&self) -> log::Level {
+    fn max_log_level(&self) -> Option<log::Level> {
```

Comment:
This can now simply be: `self.to_level_filter().to_level()` (with a correct implementation of `to_level_filter()` below).

Reply:
This wasn't implemented.